lib/logux/error_handler.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Logux
+  class ErrorHandler
```

Comment:
wdyt about renaming this class to `ErrorRenderer` or something like this to better match its purpose?